messages/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+
+class Message(models.Model):
+    user = models.ManyToManyField(User, blank=True)
```

Comment:
I'm not sure it's doing what you meant, is the same message belongs to several users?

Reply:
fixed